morris/roster.json

```diff
@@ -16,7 +16,16 @@
 			"name": "Sammie Howard"
 		},
 		{
-			"name": "Monica Segarra"
+			"name": "Monica Segarra",
+			"materials": [
+				{ 
+					"type": "cv" 
+				},
+				{
+					"type": "portfolio,
```

Comment:
@msvalen missing quotation mark after portfolio! Just add and push, no need to make a new pr